scss/components/button.scss

```diff
@@ -221,6 +224,8 @@ a.#{$block} {
   }
   //.fd-button--light
   &--light {
+    --fd-forms-height: 34px;
+    --fd-forms-height-compact: 26px;
```

Comment:
It doesn't _seem_ like this is the right place for these variables.  It seems like height-related values should be for all buttons, not just the `--light` variation.